tests/impl.h

```diff
@@ -7,532 +7,530 @@
     case it_##c:                  \
         ret = test_##c(*this, i); \
         break;
-#define INTRIN_FOREACH(TYPE)          \
```

Comment:
Why did you change many lines instead of two?

Reply:
It is caused by the formatter